orchagent/portsorch.cpp

```diff
@@ -1220,7 +1220,7 @@ void PortsOrch::doPortTask(Consumer &consumer)
             }
 
             Port p;
-            if (!getPort(alias, p))
+            if (!getPort(alias, p) && alias != "PortConfigDone")
```

Comment:
Looks like it is late in the review since it is "merged", but to not calling the getPort unnecessarily , we should use below logic:

            if (alias == "PortConfigDone" || !getPort(alias, p))
            {
                if (alias != "PortConfigDone")
                {
                    SWSS_LOG_ERROR("Failed to get port id by alias:%s", alias.c_str());
                }
            }